addons/survey/tests/test_survey_security.py

```diff
@@ -145,9 +145,9 @@ def test_access_survey_survey_user(self):
         self.env['survey.question'].create({'title': 'My Page', 'sequence': 0, 'is_page': True, 'question_type': False, 'survey_id': survey.id})
         self.env['survey.question'].create({'title': 'My Question', 'sequence': 1, 'survey_id': survey.id})
 
-        # Read: all
+        # Read: own only
         surveys = self.env['survey.survey'].search([('title', 'ilike', 'Test')])
-        self.assertEqual(surveys, self.survey | survey)
+        self.assertEqual(surveys, survey)
         surveys.read(['title'])
 
         # Write: own only
```

Comment:
Should we keep referencing "own" ? 
Maybe we should rephrase to reflect that the conditions apply  to that "Restricted to" field and not `created_uids` anymore.